package/MDAnalysis/lib/util.py

```diff
@@ -724,7 +724,15 @@ def close(self, force=False):
         .. Note:: This ``close()`` method is non-standard. ``del NamedStream``
                   always closes the underlying stream.
 
+
+        .. versionchanged:: 2.1.0
+           Calls to ``close()`` will no longer attempt to close or flush the
+           stream if :attr:`closed` is `True`.
```

Comment:
Won't it get pretty verbose if you start adding `versionchanged` directives for bug fixes as opposed to select new features or API changes?

Reply:
Yeah I can see that. I'm happy to not include that, I'm tempted to remove the entry from the changelog too if you're good with that?